src/CakeAttributesPlugin.php

```
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
+
+namespace CakeAttributes;
+
+use Cake\Core\BasePlugin;
+use Cake\Core\Configure;
+use Cake\Core\PluginApplicationInterface;
+use Cake\Routing\RouteBuilder;
+use CakeAttributes\Router\RouteProvider;
+
+/**
+ * Plugin for Attributes
+ */
+class CakeAttributesPlugin extends BasePlugin
+{
+    /**
+     * @param \Cake\Core\PluginApplicationInterface $app
+     * @return void
+     */
+    public function bootstrap(PluginApplicationInterface $app): void
+    {
+        if (!Configure::check('Cache._cake_attributes_')) {
+            // do something?
```

This is a question for you @jamisonbryant 
What would you do if a user doesn't have a attributes cache config present?
Should we throw an exception?
Should we set a default cache config?
Should it be part of the docs?

---

Create a default config if one is not set, document it.

---

Well, normally I would say we could just use `_cake_core_` but that's been recently renamed to the (more appropriate) `_cake_translations_`. Is it possible to add a new core cache config for scanned attributes?

---

Just pick a name, check if that name is configured else create one with that name.

---

I'll leave that up 2 @jamisonbryant to do 😉

---

I am creating issues for all of these, and will address shortly.